src/modules/inventory/index.ts

```diff
@@ -1,4 +1,4 @@
-import { ClientType } from '../../index';
+import { ClientType, GET_ALL_PRODUCT } from '../../index';
```

Comment:
**Suggestion(need to change:**: import this directly from queries instead of the main index, we are creating a circular import